pyqt5ac.py

```diff
@@ -167,9 +187,10 @@ def main(rccOptions='', uicOptions='', force=False, config='', ioPaths=()):
             filename, ext = os.path.splitext(basename)
 
             # Replace instances of the variables with the actual values from the source filename
-            destFilename = destFileExpr.replace('%%FILENAME%%', filename) \
-                .replace('%%EXT%%', ext[1:]) \
-                .replace('%%DIRNAME%%', dirname)
+            if 'FILENAME' in variables.keys() or 'EXT' in variables.keys() or 'DIRNAME' in variables.keys():
+                raise ValueError("Custom variables cannot be called FILENAME EXT OR DIRNAME.")
```

Comment:
When you raise this error at this point, we already made a recursive file search unnecessarily. I agree that we should fail the execution if any of these are defined even if they are not actually used in the ioPaths. Therefore I think we can make this check right after we parsed the configuration. This will definitely make the script fail fast. Could you make this check before we loop through the ioPaths?